salt/log/handlers/__init__.py

```diff
@@ -20,6 +20,13 @@
 from salt.log.mixins import NewStyleClassMixIn, ExcInfoOnLogLevelFormatMixIn
 from salt.ext.six.moves import queue
 
+import msgpack
+try:
+    import zmq
+    HAS_ZMQ = True
```

Comment:
We should really "blow up" if zmq is missing...
It's now a required piece to get logging working properly.
No?

Reply:
I had to add this for the dockermod and some ssh tests to pass.

Reply:
Ok